can/interfaces/vector/canlib.py

```diff
@@ -282,7 +282,10 @@ def __init__(
 
         # Calculate time offset for absolute timestamps
         offset = xlclass.XLuint64()
-        xldriver.xlGetSyncTime(self.port_handle, offset)
+        try:
+            xldriver.xlGetSyncTime(self.port_handle, offset)
+        except:
+            xldriver.xlGetChannelTime(self.port_handle, self.mask, offset)
```

Comment:
You could another level of try except so if both fails we can set `self._time_offset = 0` since it is not required to have absolute timestamps.

Reply:
done